src/core/agent/agent.ts

```diff
@@ -239,13 +239,11 @@ class Agent {
         Tier.low,
         agentUrls.bootUrl
       );
-      try {
-        await this.signifyClient.boot();
-      } catch (e) {
-        /* eslint-disable no-console */
-        console.error(e);
-        throw new Error(Agent.KERIA_BOOT_FAILED);
+      const bootResult = await this.signifyClient.boot();
```

Comment:
We should still keep the try catch because fetch can throw an error for other reasons (e.g. CORS etc)